packages/grid/src/features/table-ui/components/CrudStatusIndicator.tsx

```diff
@@ -0,0 +1,59 @@
+import { faExclamationTriangle } from "@fortawesome/free-solid-svg-icons/faExclamationTriangle";
+import { Icon, InputSpinner } from "@stenajs-webui/elements";
+import { Tooltip } from "@stenajs-webui/tooltip";
+import * as React from "react";
+import { CrudStatus } from "@stenajs-webui/redux";
+
+interface Props {
+  crudStatus?: CrudStatus;
+}
+
+export const hasIndicatorContent = (status: CrudStatus): boolean => {
+  const { loading, creating, deleting, updating, hasError } = status;
```

Comment:
we should probably make this type an enum or state string. Like so:

```ts
type CrudStatus = 'loading' | 'creating' | 'deleting' | 'updating' | 'hasError';
```

Reply:
it will help with the type-safety too, and minimize the amount of uncovered branches

Reply:
Yeah, maybe we can do a refactor of this later?
I only added fields on speculation.

Reply:
This is part of the crud reducer by the way.